### DISCUSSION TITLE
Popconfirm在表格中列宽不足的时候会闪烁，最近有更新吗，之前不会的

### DISCUSSION THREAD

**Discussion by whale-jung** [ant-design/ant-design#44390] [Q&A]
如题

**Comment by zombieJ** [comment]
给个 codesandbox 看看？

**Reply by whale-jung** [reply]
复现不了，https://riddle.alibaba-inc.com/riddles/new，可以看看这个地址，窗口宽度缩小到一定程度，Popconfirm会有抖动

**Reply by whale-jung** [reply]
我们平台是 Popconfirm在原来的位置和抖动之后的位置 反复闪烁
